### PR TITLE
French Translation (With the new references of code lines numbers)

### PR DESCRIPTION
**Pull request by haricot**
I just realized that the references of the line numbers is not up to date with the current code.
As a result, all current .po translations files are not up to date with the current code.

So I proceeded like this:

`djangocms-cascade $ django-admin.py makemessages -l fr - --ignore  docs/source/_static/shop_link_plugin.py - --ignore docs/source/conf.py -i examples`

( --ignore because otherwise there is this error : CommandError: Unable to find a locale path to store translations for file docs/source/_static/shop_link_plugin.py)

Maybe we need to open an issue so that we can generate a Po file update during the creation of the Package, based perhaps on this example : [Add i18n support to your Python Package Install](https://davesteele.github.io/development/2015/12/11/add-i18n-to-setup-py/).

 Or place things somewhere like this line to update .po files:
` django-admin.py makemessages   --all -i  docs/source/_static/shop_link_plugin.py -i docs/source/conf.py -i examples`

Sorry for all this, I could have seen it rather.

### CODE REVIEW

**Comment by jrief** [comment]
Thanks!
Just for curiosity, the site where you are using djangocms-cascade, is it available online?

**Comment by haricot** [comment]
This should be soon online if everything goes well, there should be several site.
I will send you the link of the first site as soon as it is done, promised.
In the future, I would like to propose you a package djangocms-bootstrap4, and plugins bs4 for djangocms-cascade. As well as an integration of djangocms-medium-editor for text fields, although this is still not very clear to me. 
At the present time, it is a mix with plugins django-cms and plugins djanocms-cascade with as constraint bs4. My goal will be to use maximum djangocms-cascade for all plugins.
I appreciate your project more and more.

**Comment by jrief** [comment]
These are good news! Bootstrap4 is definitely on my TODO list.

**Comment by haricot** [comment]
Hello Jrief,

Here is a sub-project [djangocms-bs4forcascade](https://github.com/haricot/djangocms-bs4forcascade) of djangocms-cascade.
This uses bootstrap4 alpha 6, I think that's why you had not started it, probably due to the number of versoin ^^. I can not wait

I started to interpose this in djangocms-cascade in the fork but I changed my mind again reading the 'readme' and so I intergrate it in a pluggable sub-project.

I hope this is suitable for formatting a pluggable sub-project.
All the plugins seem to work well.
The tests pass for python3 but not in python 2, for the tests _picture.py and test_image.py, I'm not on the result due to breakdown.
I have to rename certain class for tests this may be a mistake.

In short, here, I hope it will serve you.

Just for meno for translations:
If you want to use the locales from the git, you must run this to compile .mo:
`cd doc & make gettext`
